### ISSUE TITLE
How to display Runtime Statistics in Tensorboard using Estimator API in a distributed environment

### ISSUE THREAD

**Issue by Threynaud** [tensorflow/tensorflow#13594]
Hello,

I am running in the the same issue than described in this stack overflow question: https://stackoverflow.com/questions/45719176/how-to-display-runtime-statistics-in-tensorboard-using-estimator-api-in-a-distri.. 
I know that GitHub is used for features requests and bugs but this question didn't get an answer and I am not the only one running in the problem.

 This is how the doc illustrates how to add  and save Runtime statistics:
```
run_options = tf.RunOptions(trace_level=tf.RunOptions.FULL_TRACE)
run_metadata = tf.RunMetadata()
summary, _ = sess.run([merged, train_step],
                                       feed_dict=feed_dict(True),
                                       options=run_options,
                                       run_metadata=run_metadata)
train_writer.add_run_metadata(run_metadata, 'step%d' % i)
 train_writer.add_summary(summary, i)
```

Given that there is no evident way to call `sess.run` in the training phase with the Estimator API, I am genuinely wondering how to write this kind of summary... is there a workaround?

I was thinking about using a `SessionRunHook` to create something to pass to the `EstimatorSpec` but I am really not familiar with that..

**Comment by tatatodd** [comment]
@Threynaud Sorry, but we have limited resources for handling github issues, so we really do need to strictly require general questions to be handled on StackOverflow.

Closing this out.  FYI @jhseu for the StackOverflow question.

**Comment by bhack** [comment]
Can you reopen this and label it as documentation cause it is really a general request and not use case specific. 
From stackoverflow is not clear if`'ProfilerHook` is the recommended approach.

**Comment by bhack** [comment]
Also I suppose that an newbie has high probably to be exposed to high level api like keras and estimators and probably he wants to check quickly performance information on the tensorboard instead of tfprof as a first approach to profiling. So it is better to document how to quickly enable runtime statistics in TB for high level api. /cc @fchollet
